templates/default/hopsfsmount.service.erb

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description = HopsFS fuse mount
+After = syslog.target network.target remote-fs.target <%= @deps %>
+StartLimitIntervalSec=120
+StartLimitBurst=15
+
+[Service]
+User = <%= node['hops']['hdfs']['user'] %>
+PIDFile = <%= node['hops']['logs_dir'] %>/hadoop-<%= node['hops']['hdfs']['user'] %>-hopsfsmount.pid
```

Comment:
I think we may have moved all the PID files based on how Antonios is separating program files on one disk, and data files on another disk. Check with him about where PID file should go.

Reply:
nope, I talked to Antonios and PID files locations have not changed